### PR TITLE
SDK minor modifications

### PR DESCRIPTION
**Pull request by skaoth**
Noticed some small issues regarding test coverage around NPE 
and inconsistency in formatting

### CODE REVIEW

File: src/main/java/com/smartsheet/api/internal/RowColumnResourcesImpl.java [hunk 1]
```diff
@@ -85,52 +85,49 @@ public RowColumnResourcesImpl(SmartsheetImpl smartsheet) {
      * @throws SmartsheetException the smartsheet exception
      */
     public PagedResult<CellHistory> getCellHistory(long sheetId, long rowId, long columnId, PaginationParameters parameters) throws SmartsheetException {
-        String path = "sheets/" + sheetId + "/rows/"+rowId+"/columns/"+columnId + "/history";
+        String path = "sheets/" + sheetId + "/rows/" + rowId + "/columns/"+columnId + "/history";
```
Comment (ronreynolds2) [inline src/main/java/com/smartsheet/api/internal/RowColumnResourcesImpl.java]: missed one here

File: src/main/java/com/smartsheet/api/internal/SearchResourcesImpl.java [hunk 2]
```diff
@@ -66,6 +67,8 @@ public SearchResourcesImpl(SmartsheetImpl smartsheet) {
 	 * @throws SmartsheetException the smartsheet exception
 	 */
 	public SearchResult search(String query) throws SmartsheetException {
+		Util.throwIfNull(query);
```
Comment (ronreynolds2) [inline src/main/java/com/smartsheet/api/internal/SearchResourcesImpl.java]: are these 8-space indents?

File: src/main/java/com/smartsheet/api/internal/RowColumnResourcesImpl.java (hunk 1 above)
Comment (skaoth) [inline src/main/java/com/smartsheet/api/internal/RowColumnResourcesImpl.java]: lol.. yeah. Still getting to know the library

File: src/main/java/com/smartsheet/api/internal/ShareResourcesImpl.java [hunk 3]
```diff
@@ -66,22 +67,23 @@ public ShareResourcesImpl(SmartsheetImpl smartsheet, String masterResourceType)
...
+
 	public PagedResult<Share> listShares(long objectId, PaginationParameters pagination, Boolean includeWorkspaceShares) throws SmartsheetException {
 		String path = getMasterResourceType() + "/" + objectId + "/shares";
 		
 		HashMap<String, Object> parameters = new HashMap<String, Object>();
 		if (pagination != null) {
 			parameters = pagination.toHashMap();
 		}
-		if (includeWorkspaceShares == true) {
+		if (includeWorkspaceShares != null && includeWorkspaceShares == true) {
```
Comment (ronreynolds2) [inline src/main/java/com/smartsheet/api/internal/ShareResourcesImpl.java]: why is includeWorkspaceShares a Boolean to begin with?  if it were boolean there would be no problem here (also the "== true" is redundant).

File: src/main/java/com/smartsheet/api/internal/SearchResourcesImpl.java (hunk 2 above)
Comment (skaoth) [inline src/main/java/com/smartsheet/api/internal/SearchResourcesImpl.java]: I hope not! doesn't look that way in my ide. Can check

File: src/main/java/com/smartsheet/api/internal/ShareResourcesImpl.java (hunk 3 above)
Comment (skaoth) [inline src/main/java/com/smartsheet/api/internal/ShareResourcesImpl.java]: yep.. I don't know.

**Comment by timwellswa** [comment]
@skaoth you should revoke that token in your test files if you haven't already.

**Pull request merged**